lobsterpy/plotting/__init__.py

```diff
@@ -606,71 +588,182 @@ def add_all_relevant_cohps(
             label_with_count = self._insert_number_of_bonds_in_label(
                 label=bond_key, character=":", number_of_bonds=count
             )
-            if (
-                label_resolved
-            ):  # will add cohp data for each relevant bond label iteratively
-                self._cohps[label_with_count + suffix] = {}
+            # add summed cohps for each relevant bond sites
+            cohp = complete_cohp.get_summed_cohp_by_label_list(label_list=labels)
+            energies = (
+                cohp.energies - cohp.efermi if self.zero_at_efermi else cohp.energies
+            )
+            drop_down_key = plot_legend = label_with_count + suffix
+            self._cohps["All"].update(
+                {
+                    plot_legend: {
+                        "energies": energies,
+                        "COHP": cohp.get_cohp(),
+                        "ICOHP": cohp.get_icohp(),
+                        "efermi": cohp.efermi,
+                    }
+                }
+            )
+            if len(plot_data) > 1:
+                if drop_down_key not in self._cohps:
+                    self._cohps[drop_down_key] = {}
+                self._cohps[drop_down_key].update(
+                    {
+                        plot_legend: {
+                            "energies": energies,
+                            "COHP": cohp.get_cohp(),
+                            "ICOHP": cohp.get_icohp(),
+                            "efermi": cohp.efermi,
+                        }
+                    }
+                )
+
+            # will add cohp data for each relevant bond label iteratively
+            if label_resolved and not orbital_resolved:
+                if label_with_count + suffix not in self._cohps:
+                    self._cohps[label_with_count + suffix] = {}
                 for label in labels:
                     cohp = complete_cohp.get_cohp_by_label(label)
                     energies = (
                         cohp.energies - cohp.efermi
                         if self.zero_at_efermi
                         else cohp.energies
                     )
-                    outer_key = label_with_count + suffix
-                    struct = analyse.structure
-                    atom_pairs = []
-                    for site in complete_cohp.bonds[label]["sites"]:
-                        atom = site.species_string + str(struct.sites.index(site) + 1)
-                        atom_pairs.append(atom)
-                    key = "{}: {} ({} \u00c5)".format(
-                        label,
-                        "-".join(atom_pairs),
-                        str(round(complete_cohp.bonds[label]["length"], 2)),
+                    drop_down_key = label_with_count + suffix
```

Comment:
add a comment that this is a label for the drop down menu

Reply:
Done